### PR TITLE
README: remove make terraform-download

### PR DESCRIPTION
**Pull request by squat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,13 +33,9 @@ See the official Tectonic documentation:
...
 
-**Download Terraform**
+**Install Terraform**
 
-This repo a build of Terraform which includes required plugins and is pinned to a specific version. First, download the Terraform binaries:
-
-```
-$ make terraform-download
-```
+This project is built on Terraform. Download and install an [official Terraform binary](https://www.terraform.io/downloads.html) for your OS or use your favorite package manager.
```
Comment (Quentin-M) [inline README.md]: Add version?

Comment (squat) [inline README.md]: what is our minimum version requirement?

Comment (Quentin-M) [inline README.md]: =0.8.8

Comment (squat) [inline README.md]: thanks. updated.

File: README.md [hunk 2]
```diff
@@ -33,13 +33,9 @@ See the official Tectonic documentation:
...
 
-**Download Terraform**
+**Install Terraform**
 
-This repo a build of Terraform which includes required plugins and is pinned to a specific version. First, download the Terraform binaries:
-
-```
-$ make terraform-download
-```
+This project is built on Terraform and requires a version >= 0.8.8. Download and install an [official Terraform binary](https://www.terraform.io/downloads.html) for your OS or use your favorite package manager.
```
Comment (derekparker) [inline README.md]: It's actually locked to `0.8.8` exactly.

Comment (derekparker) [inline README.md]: Might want to link to https://releases.hashicorp.com/terraform/?_ga=1.63881647.1040612097.1492717955 for older releases.

Comment (squat) [inline README.md]: thanks, updated

**Pull request merged**